JSTests/wasm/gc/packed-arrays.js

```diff
@@ -0,0 +1,396 @@
+//@ runWebAssemblySuite("--useWebAssemblyTypedFunctionReferences=true", "--useWebAssemblyGC=true")
+
+import * as assert from "../assert.js";
+import { compile, instantiate } from "./wast-wrapper.js";
+
+function module(bytes, valid = true) {
```

Comment:
Since this function is just for binary tests and I think this file mainly has text tests, this function may be delete-able.

Related question: are there any cases where a binary test might be helpful? e.g., do packed types introduce any interesting new malformed binary cases?